api/triggers.yaml

```diff
@@ -7,15 +7,12 @@ components:
         triggerType:
           type: string
           enum: ["http", "grpc", "traceid"]
-        triggerSettings:
-          type: object
-          properties:
-            http:
-              $ref: "./http.yaml#/components/schemas/HTTPRequest"
-            grpc:
-              $ref: "./grpc.yaml#/components/schemas/GRPCRequest"
-            traceid:
-              $ref: "./traceid.yaml#/components/schemas/TRACEIDRequest"
+        http:
```

Comment:
There's a mismatch between the resource version and the regular version, I removed the `triggerSettings` to have them both work the same